feature-libs/product/configurators/common/src/components/config-attribute/config-attribute-types/config-attribute-multi-selection-image/config-attribute-multi-selection-image.component.html

```diff
@@ -1,5 +1,5 @@
-<div id="{{ createAttributeIdForConfigurator(attribute) }}" class="row">
-  <div
+<ul id="{{ createAttributeIdForConfigurator(attribute) }}" class="row">
+  <li
     *ngFor="let value of attribute.values; let i = index"
     id="{{ createAttributeValueIdForConfigurator(attribute, value.valueCode) }}"
     (click)="onSelect(i)"
```

Comment:
only natively interactable elements (buttons, link, etc) should be clickable. please remove `(click)` event from here, and use a button or something similar inside this `li` element

Reply:
also `(keyup)`, `role=checkbox`, `cxFocus`, `aria-checked`, and `tabindex` shouldn't be here

Reply:
Done!